next.json

```diff
@@ -691,6 +691,16 @@
             }
         ]
     },
+    "primitive topology": {
+        "category": "enum",
+        "values": [
+            {"value": 0, "name": "point"},
+            {"value": 1, "name": "line"},
```

Comment:
What do you think of using "point list", "line list" and "triangle list"?

@kainino0x WDYT too?